README.md

```diff
@@ -105,6 +105,12 @@ slotinfo continuously-for-pincode --pin_code 41105 --date 10-05-2021 --age_filte
 
 10. Run the cmd continously to check for available appointments in pin code for next 7 days after evey x(seconds) interval
 slotinfo continuously-for-pincode-next7days --pin_code 41105 --date 10-05-2021 --age_filter 18 --interval 2 --notify_on whatsapp
+
+11. Run the cmd continously to check for available appointments in pin code after evey x(seconds) interval by a vaccine type
+slotinfo continuously-for-pincode --pin_code 41105 --date 10-05-2021 --age_filter 18 --interval 2 --notify_on whatsapp --vaccine_type covishield
+
+12. Run the cmd continously to check for available appointments in pin code after evey x(seconds) interval by multiple vaccine types
+slotinfo continuously-for-pincode --pin_code 41105 --date 10-05-2021 --age_filter 18 --interval 2 --notify_on whatsapp --vaccine_type covishield --vaccine_type covaxin
```

Comment:
add these to the existing cmd's as we are not adding new cmd's here

Reply:
Should I replicate this option to each of above commands?

Reply:
yes, so that users know that this option is available. Mostly user will copy the cmds from Readme 😄